tenderly/utils/getTokenInfo.ts

```diff
@@ -0,0 +1,90 @@
+import type { evm } from '@shapeshiftoss/common-api'
+import type Web3 from 'web3'
+import type { AbiItem } from 'web3-utils'
+
+import { TOKEN_CONTRACT_ADDRESSES } from './constants'
+
+export const TOKEN_INFO_ABI: AbiItem[] = [
```

Comment:
argh, i should be using this to keep things DRY

`packages/unchained-client/src/evm/parser/abi/erc20.ts`